app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -557,7 +557,15 @@ interface MetricController {
                     isDataTelemetryEnabled,
                     isMarketingDataTelemetryEnabled
                 )
-            } else DebugMetricController()
+            } else {
+                // todo switch it back to normal
+                ReleaseMetricController(
```

Comment:
We should revert this part before landing.

fwiw, you can do `git add <path-to-file>` before committing to prevent adding changes like this or, if you want more granularity than per file, `git add -p <path-to-file>` to be able to select individual lines to add.

I can also help you undo this change using git commands if you'd like!

Reply:
Ah, thanks! I think I figured it out now. Didn't know about the -p option that's very neat!